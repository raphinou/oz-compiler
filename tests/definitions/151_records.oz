% auto number features of records if necessary (desugar step)
local
   R = rec(f1 b:f2 c:f3)
   R2= rec(a b c)
in
   {Show R.1}
   {Show R.b}
   {Show R2.1}
   {Show R2.2}
   {Show R2.3}
end

