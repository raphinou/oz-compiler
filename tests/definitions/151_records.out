f1
f2
a
b
c
